Gemfile

```diff
@@ -165,6 +165,10 @@ group :development do
 
   # When we want to squash migrations
   gem 'squasher'
+
+  # profiling
+  gem 'rack-mini-profiler', require: false
+  gem 'stackprof'
```

Comment:
@sojan-official I add this gem, gives neat profiles like this 
<img width="929" alt="image" src="https://user-images.githubusercontent.com/18097732/227889924-a201c5b4-189b-4a2d-a95e-24e38f61b951.png">

Is it worth keeping it? I can remove it, it's very simple to add whenever I need to test

Reply:
should be good to go for development group